test/unit/org/apache/cassandra/index/sai/cql/AbstractQueryTester.java

```diff
@@ -80,8 +80,12 @@ public static List<Object[]> params() throws Throwable
     {
         List<Object[]> scenarios = new LinkedList<>();
 
-        for (Version version : List.of(Version.AA, Version.CA))
+        for (Version version : Version.ALL)
         {
+            // Excluding BA from the version matrix as it has never seen production deployment as a write target
```

Comment:
Just for your understanding, it was in prod, but we've upgraded all versions and do not intend to use again. This was the HNSW index version. Other than HNSW vs diskann, CA and BA are the same.